--- /opt/odoo-helps/jquery.js	2021-11-05 13:02:03.489269866 +0200
+++ /usr/lib/python3/dist-packages/odoo/addons/web/static/src/lib/jquery/jquery.js  2021-01-16 12:31:15.371057945 +0200
@@ -4666,11 +4666,8 @@
 
                 // Find delegate handlers
                 // Black-hole SVG <use> instance trees (#13180)
-                // ODOO CHANGE: cherry-picking https://github.com/jquery/jquery/commit/c82a6685bb9
-                // Support: Firefox<=42+
-                // Avoid non-left-click in FF but don't block IE radio events (#3861, gh-2343)
-                if ( delegateCount && cur.nodeType &&
-                        ( event.type !== "click" || isNaN( event.button ) || event.button < 1 ) ) {
+                // Avoid non-left-click bubbling in Firefox (#3861)
+                if ( delegateCount && cur.nodeType && (!event.button || event.type !== "click") ) {
 
                         /* jshint eqeqeq: false */
                         for ( ; cur != this; cur = cur.parentNode || this ) {
